libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/MapRouteArrow.java

```diff
@@ -123,10 +124,16 @@ void addUpcomingManeuverArrow(RouteProgress routeProgress) {
   }
 
   void updateVisibilityTo(boolean visible) {
-    for (Layer layer : arrowLayers) {
-      String targetVisibility = visible ? VISIBLE : NONE;
-      if (!targetVisibility.equals(layer.getVisibility().getValue())) {
-        layer.setProperties(visibility(targetVisibility));
+    Style style = mapboxMap.getStyle();
+    if (style != null) {
```

Comment:
```suggestion
    mapboxMap.getStyle()?.let { style ->
```
wdyt?

Reply:
I deliberately used the null-checked one. We don't want to wait for style to be available - if it's not there, we should just skip. The upstream logic takes care of reloading the class when a new style is available.

Reply:
Oh, you suggested `let`. This is a java class :)

Reply:
Oh, sorry, my fault, it is Java :)